cfgov/cfgov/settings/test.py

```diff
@@ -106,7 +106,4 @@ def __getitem__(self, item):
 
     MIGRATION_MODULES = _NoMigrations()
 
-for search_backend_settings in WAGTAILSEARCH_BACKENDS.values():
-    search_backend_settings["AUTO_UPDATE"] = False
```

Comment:
Lordy.

Reply:
I actually wrestled with leaving this in. I added it in https://github.com/cfpb/consumerfinance.gov/commit/e4127d2855f2029317ce56766e29350a14448864 to speed up tests. Unfortunately leaving it in would require leaving in the definition of `WAGTAILSEARCH_BACKENDS` in the base settings, so I opted for simpler code over a minor speedup.